lab07.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+function StoreCalculation(name, mincust, maxcust, avcook){
+  this.name = name;
+  this.mincust = mincust;
+  this.maxcust = maxcust;
+  this.avcook = avcook;
+  this.results = [];
+  this.total = 0;
+  this.time = ['6am', '7am', '8am', '9am' , '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '7pm', '8pm'];
+
+}
+
+StoreCalculation.prototype.avCust = function () {
+  return Math.floor((Math.random() * (this.maxcust - this.mincust + 1)) + this.mincust);
+
+};
+
+StoreCalculation.prototype.storeMath = function(){
+  for (var i = 0; i < 15; i++){
+    var averageCalc = Math.round(this.avcook * this.avCust());
+    this.results.push(averageCalc);
+    this.total += this.results[i];
+
+  }
+  console.log (this.results);
+  console.log(this.total);
+};
+
+StoreCalculation.prototype.storeTable = function () {
+  var table = document.createElement('table');
```

Comment:
It looks like you're creating a new table for each store. We only want one table with all the stores.